### PR TITLE
Step Enhancements

### PR DESCRIPTION
**Pull request by ROODAY**
Resolves issues #97 and #100. 

The former was solved using the package [readline-sync](https://www.npmjs.com/package/readline-sync).  
The latter was solved by just splitting program.step by spaces.  
The two were combined with a few forEach loops.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,19 +30,19 @@ require('fs').access(program.image, function(err){
   if(err) exit("Can't read file.")
 });
 
-// User must input a step.
-if(!program.step || !sequencer.modulesInfo().hasOwnProperty(program.step))
+// User must input a step. If steps exist, check that every step is a valid step.
+if(!program.step || !program.step.every(function(step){return sequencer.modulesInfo().hasOwnProperty(step)}))
```
Comment (ccpandhare) [inline index.js]: Sorry for being choosy here, but could you make this a bit more legible? Thanks!

Comment (ROODAY) [reply index.js]: Should I abstract that line to a function?

**Comment by jywarren** [comment]
This looks good -- but can you submit a README addition for the CLI syntax for multiple steps? Thanks!

**Comment by ccpandhare** [comment]
Looks good to merge!

**Comment by jywarren** [comment]
+1!!! 🎉

**Pull request merged**